AlignmentPalette/Alignment.glyphsPalette/Contents/Resources/plugin.py

```diff
@@ -155,16 +157,18 @@ def overshootsOfLayer( self, layer ):
 						if zone.size > 0 and node2.y >= zone.position and node2.y <= zone.position + zone.size:
 							overshoot = node2.y - height
 							existingOvershoot = overshoots[index][1]
-							if overshoot > existingOvershoot:
-								overshoots[index][1] = overshoot
+							if not existingOvershoot is None:
```

Comment:
Why do we need to check this? If we initialise all as 0 then it will never be None.

Reply:
You are right. This is residue from a previous fix, because it crashed because it was sometimes `None`.